packages/manager/modules/overthebox/src/overthebox/actions/index.js

```diff
@@ -0,0 +1,17 @@
+import angular from 'angular';
+
+import ngTranslateAsyncLoader from '@ovh-ux/ng-translate-async-loader';
+import uiRouter from '@uirouter/angularjs';
```

Comment:
As those are peers, maybe we should prefer 
```suggestion
import '@ovh-ux/ng-translate-async-loader';
import  '@uirouter/angularjs';
```
and
```suggestion
angular
  .module(moduleName, ['ngTranslateAsyncLoader', 'ui.router', 'ovh-api-services'])
```